components/avatar/avatar.vue

```diff
@@ -6,7 +6,17 @@
   >
     <div
       ref="canvas"
-      :class="[canvasClass, 'd-avatar__canvas', { 'd-avatar--image-loaded': imageLoadedSuccessfully }]"
+      :class="[
+        canvasClass,
+        'd-avatar__canvas',
+        {
+          'd-avatar--image-loaded': imageLoadedSuccessfully,
+          'd-avatar--clickable': clickable,
+        },
+      ]"
+      :tabindex="clickable ? 0 : -1"
+      :role="clickable ? 'button' : 'presentation'"
```

Comment:
Can we change the actual element instead of just setting role? Can we have some sort of indicator on hover to show it is clickable? Maybe overlay it with a dark color with some opacity?

Reply:
What element should I use instead, a button?
Maybe I can show the focus ring on hover too, I was using  `cursor: pointer` to show it is clickable on hover but maybe the opacity would work better.

@francisrupert @fede-dp please help us on this decision.

Reply:
Yeah it should be button I think

Reply:
updated